docs-src/app-dev-context/skip-time-set-up.md

```diff
@@ -0,0 +1,10 @@
+---
+id: skip-time-set-up
+title: Set up Time Skip
```

Comment:
```suggestion
title: Set up time skipping
```
Doesn't much matter at this time, but just in case this gets reused in a different context.